osu.Game/Screens/Select/Options/BeatmapOptionsButton.cs

```diff
@@ -31,6 +31,31 @@ public class BeatmapOptionsButton : OsuClickableContainer
         private readonly OsuSpriteText secondLine;
         private readonly Container box;
 
+        private const float disabled_alpha = 0.5f;
+
+        private bool disabled;
+
+        public bool Disabled
+        {
+            get => disabled;
```

Comment:
This class already inherits `OsuClickableContainer`, which provides an `Enabled` property. Please use that instead of inventing another field that's opposite to it.